library/core/src/num/f64.rs

```diff
@@ -719,6 +719,62 @@ impl f64 {
         intrinsics::minnumf64(self, other)
```

Comment:
Consider, while you're in the file, adding IEEE references to these functions too -- maybe something like
```
/// This follows the `minNum` semantics specified in IEEE 754-2019.
```

Reply:
Per [`llvm.minnum.*`](https://llvm.org/docs/LangRef.html#llvm-minnum-intrinsic) this is not quiet true: 
> Follows the IEEE-754 semantics for minNum, except for handling of signaling NaNs. This match’s the behavior of libm’s fmin.

> Unlike the IEEE-754 2008 behavior, this does not distinguish between signaling and quiet NaN inputs. If a target’s implementation follows the standard and returns a quiet NaN if either input is a signaling NaN, the intrinsic lowering is responsible for quieting the inputs to correctly return the non-NaN input (e.g. by using the equivalent of llvm.canonicalize).

Adding the first citation seems better (with the mention of the 2008 revision, as `minNum` and `maxNum` were removed from the 2019 revision).

Reply:
Done.